### DISCUSSION TITLE
Unable to compile Carla "make PythonAPI "

### DISCUSSION THREAD

**Discussion by Abhishek2011992** [carla-simulator/carla#4339] [Q&A]
i have visual studio 2017 installed in my PC but i get below error

![image](https://user-images.githubusercontent.com/67043133/123505913-25e6b780-d67f-11eb-9f0d-e758355d77ce.png)

**Comment by corkyw10** [comment]
Hello, can you tell me what version of CARLA you are building and provide a link to the documentation you are using?
Have you checked that the whole CARLA repo was downloaded correctly to your computer?
